sdk/eventhub/Azure.Messaging.EventHubs.Processor/samples/Sample02_EventProcessorConfiguration.md

```diff
@@ -0,0 +1,246 @@
+# Event Processor Configuration
+
+The `EventProcessorClient` supports a set of options to configure its behavior.  In addition to influencing processing behavior, the options also support configuration focused on communication with the Event Hubs service and core functionality.  This sample demonstrates some of the options for customizing the configuration for the core areas.  Configuration of processing functionality will be discussed in the samples dedicated to that feature.
```

Comment:
nit: This seems to stand better on its own rather than talking a lot about other options not discussed in this sample.
```suggestion
The `EventProcessorClient` supports a set of options to configure many aspects of its core functionality including how it communicates with the Event Hubs service.  This sample demonstrates some of these options.
```